src/dbcp/transform/ballot_ready.py

```diff
@@ -65,7 +67,7 @@ def _normalize_entities(ballot_ready: pd.DataFrame) -> dict[str, pd.DataFrame]:
     # and the future/current one are reflected in the data.
 
     # check if position_id is unique
-    br_positions = ballot_ready.drop_duplicates(subset=position_fields)
+    br_positions = ballot_ready.drop_duplicates(subset=position_pk_fields)
     is_duplciate_position = br_positions.position_id.duplicated(keep=False)
```

Comment:
I don't understand this change. We drop duplicates based on `position_id` then look for duplicates of `position_id`?

Reply:
Oh i'm a ding dong. Thanks for the catch. I changed it to drop using all position columns.